waspc/data/Generator/templates/server/rollup.config.js

```diff
@@ -17,12 +19,20 @@ function createBundle(inputFilePath, outputFilePath) {
       sourcemap: true,
     },
     plugins: [
+      resolve(),
+      alias({
+        entries: [
+          {=# aliases =}
+          { find: '{= find =}', replacement: '{= replacement =}' },
+          {=/ aliases =}
+        ]
```

Comment:
NOTE: Just like `paths` in `sdk/tsconfig`, this field's value is an empty array when the user doesn't define any aliases.